### DISCUSSION TITLE
Error on sending mail with Office365 mailbox

### DISCUSSION THREAD

**Discussion by nessor** [lukevella/rallly#517] [Q&A]
Hi guys!

I've got an error when I'm trying to send emails with an office365 mailbox:

```
[Error: C0E73597AA7F0000:error:0A00010B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:355:
] {
  library: 'SSL routines',
  reason: 'wrong version number',
  code: 'ESOCKET',
  command: 'CONN'
}
```

Has anyone got this error before, does anyone know how I can solve the problem/what is causing the problem?
At first glance it looks like the SSL handshake didn't work because of wrong version numbers.

Looking forward to answers!

**Comment by ggilestro** [comment]
Try setting `SMTP_SECURE=false `
